lib/Runtime/Library/JavascriptRegularExpression.h

```diff
@@ -51,8 +51,15 @@ namespace Js
 
         Var GetOptions();
 
-        inline void SetPattern(UnifiedRegex::RegexPattern* pattern);
-        inline void SetSplitPattern(UnifiedRegex::RegexPattern* splitPattern);
+        void SetPattern(UnifiedRegex::RegexPattern* pattern)
```

Comment:
Yeah, i think its fine too but it's probably unnecessary. IIRC, inline is implied for member functions defined within a class definition